lib/routes.js

```diff
@@ -24,6 +24,33 @@ module.exports = function(self, options) {
     });
   });
 
+  self.route('post', 'revert', function(req, res) {
+    var id = self.apos.launder.id(req.body.id);
+    return self.revert(req, id, function(err, results) {
+      if (err) {
+        return res.send({
+          status: (typeof (err) === 'string') ? err : 'error'
+        });
+      }
+
+      // On success, grab current version of reverted doc and include url for client-side redirect
+      return self.apos.docs.find(req, { _id: id }).toArray(function(err, docs) {
+        var url;
+        if (err) {
+          console.log('Error retrieving reverted document', err);
```

Comment:
We do not use console.log server-side except during development, call `self.apos.utils.error`. `eslint` should have called this to your attention?

Reply:
OK the rule about not using console is not in this project yet. We should bring that over from the apostrophe project. Please use `self.apos.utils.error`, `self.apos.utils.info`, etc.

Reply:
done

Reply:
done